docs/source/groupby-api.rst

```diff
@@ -0,0 +1,19 @@
+API
```

Comment:
Probably after the "Series Methods", but before the other functions section. Again, just a slight preference - reorganizing later if needed is easy once the content is created.